test/unit/ssv-registry.js

```diff
@@ -39,18 +39,75 @@ describe("Validators", () => {
         // await deployedRegistryContract.createGroup([1,2,3,4]);
 
         await deployedRegistryContract.deposit("100000000000");
-
+        console.log(`[BLOCK] ${await blockNumber()}`)
+        console.log('> deposit');
+        console.log('> 1 operator balance', await deployedRegistryContract.test_getOperatorBalance(1), 'index', await deployedRegistryContract.test_operatorCurrentIndex(1));
+        console.log('> 2 operator balance', await deployedRegistryContract.test_getOperatorBalance(2), 'index', await deployedRegistryContract.test_operatorCurrentIndex(2));
+        // console.log('> 3 operator balance', await deployedRegistryContract.test_getOperatorBalance(3), 'index', await deployedRegistryContract.test_operatorCurrentIndex(3));
+        // console.log('> 4 operator balance', await deployedRegistryContract.test_getOperatorBalance(4), 'index', await deployedRegistryContract.test_operatorCurrentIndex(4));
         // validator 1
-        await expect(await deployedRegistryContract.registerValidator(
-                [1,2,3,4],
-                validatorPK,
-                sharePKs.slice(0, 4),
-                encryptedShares.slice(0, 4),
-                "10000"
-            ))
-            .to.emit(deployedRegistryContract, 'ValidatorAdded');
-            //.withArgs(validatorPK);
-
+        await progressBlocks(97);
+        const resultRegister = (await (await deployedRegistryContract.registerValidator(
+            [1, 2],
```

Comment:
@vadiminc 
a. why only 2 operators
b. there are no check operators that exist?
c. please in tests have 2 registered validator cases:
1. (new collection and group creation) - most expensive
2. validator to the same/existing group - least expensive